VAMobile/src/translations/en/common.json

```diff
@@ -863,7 +863,7 @@
   "howWillYou.useInfo.2.a11yLabel": "If you’re enrolled in V-A health care, we’ll send your prescriptions to your mailing address. Your health care team may also use this information to contact you.",
   "militaryInformation.title": "Military information",
   "militaryInformation.branch": "{{branch}}",
-  "militaryInformation.history": "{{begin}} - {{end}}",
```

Comment:
This character looks the same in my browser, but I guess it's actually a different character? Is it supposed to be a longer em dash like &mdash; ?

Reply:
Yup. That's the difference.